pandas/tests/extension/base.py

```diff
@@ -0,0 +1,330 @@
+import numpy as np
```

Comment:
Should we put a module docstring here at the top saying something in the sense that those tests are used by external extension arrays to test themselves, so that those tests and their names should not be changed lightly for the sake of changing, only if really needed?